Clinical-Genomics/scout
# Bypass CORS check in igv.js using cors-anywhere service and add IGV.js tracks documentation

Allow igv.js to use any cloud resource, even if the cloud storage provider doesn't support the strict igv.js CORS policy.

**How to test**:
1. Add this track configuration to the demo config file:
**Note that the chromosome build is "38"**. This way you could test the PR by looking at one MT variant in IGV (uses chromosome build 38)

```
CLOUD_IGV_TRACKS = [
    {
        "name": "custom_public_bucket",
        "access": "public",
        "tracks": [
            {
                "name": "dbVar Pathogenic or Likely Pathogenic",
                "type": "variant",
                "format": "vcf",
                "build": "38",
                "url": "https://s3-eu-west-1.amazonaws.com/pfigshare-u-files/25777457/GRCh37.variant_call.clinical.pathogenic_or_likely_pathogenic.vcf.gz",
                "indexURL": "https://s3-eu-west-1.amazonaws.com/pfigshare-u-files/25777460/GRCh37.variant_call.clinical.pathogenic_or_likely_pathogenic.vcf.gz.tbi",
            }
        ],
    },
]
```

2. from master branch, launch demo server (`scout --demo serve`) and go to this MT variant: http://localhost:5000/cust000/643594/f75deb14514df16195735a4466eae2d0
3. Click on the igv browser (make sure dbVar track is selected)
4. Make sure that the dbVar track doesn't load
5. Switch to this branch and repeat
6. Now the dbVar track should be shown


**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
